.github/workflows/php.yml

```diff
@@ -0,0 +1,50 @@
+name: PHP Composer
+
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    branches: [ "2.5" ]
```

Comment:
what does this mean/do?

Reply:
Runs the CI when any PRs are made against the `2.5` branch.

Reply:
ahh makes sense